### PR TITLE
Initial structure for packaging re-usable indy-cat modules

### PR DESCRIPTION
**Pull request by ianco**
Initial structure for TheOrgBook's api_v2, to be packaged as a Django application.

(api_v2 not yet included!)

### CODE REVIEW

File: credential-registry/server/django-icat-api/LICENSE [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+      To apply the Apache License to your work, attach the following
+      boilerplate notice, with the fields enclosed by brackets "{}"
+      replaced with your own identifying information. (Don't include
+      the brackets!)  The text should be enclosed in the appropriate
+      comment syntax for the file format. We also recommend that a
+      file or class name and description of purpose be included on the
+      same "printed page" as the copyright notice for easier
+      identification within third-party archives.
+
+   Copyright {yyyy} {name of copyright owner}
```
Comment (nrempel) [inline credential-registry/server/django-icat-api/LICENSE]: I think we should find out what the correct attribution for this licence is.

Also, should we have the licence only in the root of repo or also in the sub-projects? (/agent would also need to get one)

File: credential-registry/server/django-icat-api/README.rst [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+    path("api/v2/", include("api_v2.urls")),
+
+3. Move the api_v2 folder (remove from TheOrgBook repo) into this directory
+
+4. Run "python setup.py sdist" to create a distribution
+
+5. Include the following in your Docker build pipeline to build the OrgBook image:
+
+build-api() {
```
Comment (nrempel) [inline credential-registry/server/django-icat-api/README.rst]: I'm not too familiar with .rst but this doesn't seem to be rendering properly for me

File: credential-registry/server/icat-py-build.sh [hunk 3]
```diff
@@ -0,0 +1,7 @@
+
```
Comment (nrempel) [inline credential-registry/server/icat-py-build.sh]:
```suggestion
#!/bin/bash
```

File: credential-registry/server/django-icat-api/LICENSE (hunk 1 above)
Comment (ianco) [reply credential-registry/server/django-icat-api/LICENSE]: Good point I just copied this from one of our other repos.  I'll look into it.

The 2 python packages may need their own license depending on how we distribute the code (e.g. if we distribute through PyPI)

File: credential-registry/server/django-icat-api/README.rst (hunk 2 above)
Comment (ianco) [reply credential-registry/server/django-icat-api/README.rst]: Fixed

**Pull request merged**